packages/backend/src/open_payments/grant/service.test.ts

```diff
@@ -137,6 +137,19 @@ describe('Grant Service', (): void => {
         })
       ).resolves.toBeUndefined()
     })
+
+    test('cannot store grant with misformatted management url', async (): Promise<void> => {
+      const options: CreateOptions = {
+        accessToken: 'aq1sw2de3fr4',
+        managementUrl: '',
```

Comment:
Do we want all of these to fail? Because if so, I'll have to rewrite the checking function. Right now, it only checks that it can parse some sort of id but does not care about the path. 
Actually, it only checks that there is any string, which definitely needs updating.